Pod/Classes/ios/NYTPhotosViewController.h

```diff
@@ -207,6 +207,11 @@ extern NSString * const NYTPhotosViewControllerDidDismissNotification;
  */
 - (void)photosViewController:(NYTPhotosViewController *)photosViewController actionCompletedWithActivityType:(NSString * _Nullable)activityType;
 
+/**
+ *  Dismisses `NYTPhotosViewController`. Should only be used in cases where user interaction of a button in the captionView requires a different viewController to be modal.
+ */
+- (void)forceDismiss;
```

Comment:
Now that dismissal is being exposed publicly, it would be really nice to provide an option on whether to animate, and the option to pass a completion block, mirroring the ones available on `dismissViewControllerAnimated:completion:`.

@dzlobin @cdzombak What do you think?